runner_manager/jobs/runner_groups.py

```diff
@@ -0,0 +1,18 @@
+from runner_manager.dependencies import get_settings
+from runner_manager.models.runner_group import RunnerGroup
+from runner_manager.models.settings import Settings
+
+
+def create_runner_group(settings: Settings = get_settings()):
+    runner_groups_configs = settings.runner_groups
+    existing_groups = RunnerGroup.find().all()
+    for runner_group_config in runner_groups_configs:
+        if runner_group_config.name not in [group.name for group in existing_groups]:
+            runner_group = RunnerGroup(**runner_group_config.dict())
+            runner_group.save()
+        else:
+            runner_group = RunnerGroup.find(RunnerGroup.name == runner_group_config.name).first()
+            runner_group.update()
+            existing_groups.remove(runner_group)
+    for runner_group in existing_groups:
+        runner_group.delete()
```

Comment:
Couple of things missing, we can work on it as needed.

- [ ] Add a `@classmethod` into the RunnerGroup called `create_group` or whatever name fits that will:

  - Take `cls` and `GitHub` client as parameter
  ```python
  @classmethod
  def create_group(cls, github: GitHub) -> "RunnerGroup":
      
  ```
  - Create a runner_group on github (or/and check if the runner group does not exist already).
  - Set the `RunnerGroup.id` to the retrieved id from the group creation.
  - `.save()` the runner
  - return the created runner group: `return cls.save()` works.
- [ ] Tests
- [ ] Modify inputs of method, with `settings: Settings, github: GitHub`
- [ ] When deleting the runner group we should delete existing runners (if any) and delete the group from github. Another class method can be created for it.